operator/data/examples/multicluster/values-istio-multicluster-primary.yaml

```diff
@@ -2,31 +2,33 @@ apiVersion: install.istio.io/v1alpha1
 kind: IstioOperator
 spec:
   values:
-    gateways:
-      istio-ingressgateway:
-        env:
-          ISTIO_META_NETWORK: "network1"
+    # selfSigned is required if Citadel is enabled, that is when values.global.istiod.enabled is false.
+    security:
+      selfSigned: false
+
     global:
-      mtls:
-        enabled: true
-      controlPlaneSecurityEnabled: true
-      proxy:
-        accessLogFile: "/dev/stdout"
+      multiCluster:
+        clusterName: main0
```

Comment:
can you comment where this comes from in the yaml? It needs to match your secret right?

Reply:
added a comment, this is just the cluster name user specifies.  It will be used as part of the name in the secret also, e.g. istio-remote-secret-main0.

BTW detailed docs are here - https://istio.io/docs/setup/install/multicluster/shared/#main-cluster